Jenkinsfile_k8s

```diff
@@ -7,4 +9,5 @@ buildDockerAndPublishImage('jenkins-weekly', [
     targetplatforms: 'linux/amd64,linux/arm64',
     nextVersionCommand: 'echo "$(jx-release-version -next-version=semantic:strip-prerelease)-$(grep "FROM jenkins" Dockerfile | cut -d: -f2 | cut -d- -f1)"',
     dockerBakeFile: 'docker-bake.hcl',
+    dockerBakeTarget: 'weekly-ci',
```

Comment:
```suggestion
```

Added to test https://github.com/jenkins-infra/pipeline-library/pull/822 (with success), not needed anymore.